src/Gui/Forms/MainFormInteractor.cs

```diff
@@ -219,26 +220,25 @@ public IMainForm MainForm
             get { return form; }
         }
 
-        public void OpenBinary(string file)
+        public void OpenBinary(string file, string outputDir)
         {
-            OpenBinary(file, (f) => pageInitial.OpenBinary(f));
+            OpenBinary(file, outputDir, (f) => pageInitial.OpenBinary(f, outputDir));
         }
 
         /// <summary>
         /// Master function for opening a new project.
         /// </summary>
         /// <param name="file"></param>
         /// <param name="openAction"></param>
-        public void OpenBinary(string file, Func<string,bool> openAction)
+        public void OpenBinary(string file, string outputDir, Func<string,bool> openAction)
         {
             try
             {
                 CloseProject();
                 SwitchInteractor(InitialPageInteractor);
-                if (openAction(file))
-                {
-                    ProjectFileName = file;
-                }
+                openAction(file);
```

Comment:
If a file could not be opened we should not be setting the name of the opened file property.